mysql-test/suite/innodb/t/doublewrite.test

```diff
@@ -48,130 +48,32 @@ insert into t1 values(4, repeat('-',12));
 insert into t1 values(5, repeat('.',12));
 commit work;
 
+flush table t1 for export;
+unlock tables;
+
 --echo # ---------------------------------------------------------------
 --echo # Test Begin: Test if recovery works if first page of user
 --echo # tablespace is full of zeroes.
 
-select space from information_schema.innodb_sys_tables
-where name = 'test/t1' into @space_id;
+select space into @space_id from information_schema.innodb_sys_tables
+where name = 'test/t1';
 
 begin;
 insert into t1 values (6, repeat('%', 12));
---echo # Ensure that dirty pages of table t1 are flushed.
-set global innodb_buf_flush_list_now = 1;
 
---source ../include/no_checkpoint_start.inc
+set global innodb_log_checkpoint_now = 1;
 
 --echo # Make the first page dirty for table t1
 set global innodb_saved_page_number_debug = 0;
 set global innodb_fil_make_page_dirty_debug = @space_id;
 
---echo # Ensure that dirty pages of table t1 are flushed.
-set global innodb_buf_flush_list_now = 1;
+SET GLOBAL innodb_max_dirty_pages_pct_lwm=0.0;
+SET GLOBAL innodb_max_dirty_pages_pct=0.0;
 
---let CLEANUP_IF_CHECKPOINT=drop table t1;
---source ../include/no_checkpoint_end.inc
 disconnect stop_purge;
-
---echo # Make the first page (page_no=0) of the user tablespace
---echo # full of zeroes.
---echo #
---echo # MDEV-11623: Use old FSP_SPACE_FLAGS in the doublewrite buffer.
```

Comment:
The commit message fails to mention why this section of the test was removed. The commit message only says "what" but not "why".

Is this test meaningless only for 10.6, due to 86dc7b4d4cfe15a2d37f8b5f60c4fce5dba9491d?

Reply:
we skip the page if we have flag mismatch during `recv_dblwr_t::find_page()` ?
`
commit dbba1bb1c3522090f9a3bc54c107d684bfaed4ab (origin/st-10.5-MDEV-32144, st-10.5-MDEV-32144)
Author: Thirunarayanan Balathandayuthapani <thiru@mariadb.com>
Date:   Thu Oct 19 13:11:20 2023 +0530

    MDEV-31851   After crash recovery, undo tablespace fails to open
    
    recv_recovery_from_checkpoint_start(): InnoDB should add the
    redo log block header + trailer size while checking the log
    sequence number in log file with log sequence number in the
    system tablespace first page.
`

Reply:
Please add the explanation to this commit message, so that it can be easily found in the history without looking up this pull request.